core/src/components/ripple-effect/test/basic/ripple-effect.e2e.ts

```diff
@@ -55,10 +57,7 @@ const verifyRippleEffect = async (page: E2EPage, selector: string) => {
     await page.mouse.down();
   }
 
-  // Waits for the ripple effect to be added
-  await page.waitForSelector(`${selector}.ion-activated`);
+  await page.waitForSelector('.ion-activated');
```

Comment:
Is this needed? The `expect` below should wait for a few seconds automatically if it doesn't initially pass, AFAIK.

Reply:
Yeah this was needed. Otherwise I found the assert to be flaky, where it would evaluate the class list before `ion-activated` was applied.

Reply:
Makes sense 👍